recipes/app3.rb

```diff
@@ -138,3 +138,35 @@
   ]
   sensitive true
 end
+
+[
+  ['etherpad-lite.osuosl.org', 8085, 'osuosl/etherpad', data_bag_item('etherpad', 'mysql_creds_osl')],
+  ['etherpad-snowdrift.osuosl.org', 8086, 'osuosl/etherpad-snowdrift', data_bag_item('etherpad', 'mysql_creds_snowdrift')],
+].each do |hostname, port, image, creds|
+  # Check if attribute is set for testing
+  db_host = if node['osl-app'].attribute?('db_hostname')
+              node['osl-app']['db_hostname']
+            else
+              creds['db_hostname']
+            end
```

Comment:
This reads better I think
```suggestion
  db_host = if node['osl-app'].attribute?('db_hostname') ? node['osl-app']['db_hostname'] : creds['db_hostname']
```